### DISCUSSION TITLE
Can't Scroll on Local Screen Except by Scroll Bar

### DISCUSSION THREAD

**Discussion by feebb** [rern/rAudio#1562] [Bug Reports]
Since the switch from chromium to Firefox for the local browser, I haven't been able to scroll by touching anywhere on the screen; I can only scroll using a tiny scroll bar on the far right of the screen. Can this be fixed, or should I just use an earlier version of rAudio?

Thanks!

**Comment by one-step-behind** [comment]
@feebb I had the same problem with a DSI Touchscreen hooked onto my RPi 3B+.
Solution: edit `/etc/security/pam_env.conf` and add these lines:
```bash
# Enable touch-scrolling in Firefox
MOZ_USE_XINPUT2 DEFAULT=1
```
Reload browser/UI afterwards (or reboot).

source: https://superuser.com/a/1485044

**Comment by feebb** [comment]
That worked perfectly!
Thanks!
